purchase_request_to_rfq_order_approved/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License LGPL-3
+
+Purchase Request to RFQ Order Approved
+======================================
+
+This module computes the new PO state 'Approved' related to a Purchase
+Request Line to display it in the Purchase Request Line tree view and adds a
+'Purchase Approved' filter.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/142/9.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/purchase-workflow/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+Eficent. http://www.eficent.com/
+
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
```

Comment:
https. See https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst. Use the maintainer section, as it is provided in that link.